### PR TITLE
Collected Goggins Comments on Social Currency Metric System

### PR DESCRIPTION
**Pull request by sgoggins**
It may be easier to add comments as reviews on this pull request than process each of my issue comments. Its at least consolidated and seems like an easier way to converse about the thoughts I shared. Up to you. I don't expect it to be merged.

### CODE REVIEW

**Comment by ElizabethN** [comment]
Just gonna mention @GeorgLink @samanthavenialogan @valeriocos @ria18405 for their comments. I'm not sure who else worked on the development of this metric, but feel free to include anyone else that might have missed these comments. :+1:

**Comment by samanthavenialogan** [comment]
Soooo to start with...
Can we have permission to quote this until everyone's sick of hearing it? 

"SCMS has great potential for providing widespread benefit to countless open source communities, and the effort placed into its development to date warrants recognition."

Just wow, that comment blew us away. Anyway, though, we took some time to think about the future of the SCMS with CHAOSS, and we love all of the annotations and changes we're seeing.  

While fitting the SCMS into a "metric" rather than the "system" we originally defined it as was shoving it into a square hole, it was also forming the system into something new and we are excited to see where it goes.  Additionally, the social currency metrics we identified to build the system were intended to make concrete measurements out of vague concepts. This new opportunity seems to be the next step in that iterative process and this time we're grateful we have a community to help us do it. Making these more definable was always the goal, and we'd love it if we can follow that process with CHAOSS to make the system more useful.  

So, in short, yes, absolutely, we are in alignment with this and in agreement that moving the SCMS from a metric to a group of metrics in a defined system is the way to go.  And we are over the moon excited to hear that has the potential to become a full project in its own right.  

Both Dylan and I have been thrilled to see this system we've dedicated so much to, get picked up by a beautiful and brilliant community, and we can't wait to learn more about what this will become and how we can foster the SCMS’ growth in the CHAOSS community. 

We do, however, want to ask what this change would mean for the SCMS' current status as a metric?
